packages/ui/src/components/editor/input/Vector3/index.tsx

```diff
@@ -46,18 +46,18 @@ export const Vector3Scrubber = ({ axis, onChange, onPointerUp, value, children,
   const color = (() => {
     switch (axis) {
       case 'x':
-        return 'red-500'
+        return '[#375DAF]'
       case 'y':
-        return 'green-400' // must be fushsia-400 , but these colors doesnt show up
+        return '[#A24482]'
       case 'z':
-        return 'blue-400' //must be teal-400 , but this color doesnt show up
+        return '[#8261D2]'
```

Comment:
![image](https://github.com/user-attachments/assets/9b378bcf-eb21-44d2-8981-13b058b58d65)

The colors of the axes are somewhat different in values in design. (https://www.figma.com/design/DXwFHGa5g0peqqSjyjpxQz/Studio-Design?node-id=169-17899&t=VldGUxtgFDVpAXQr-0)

Reply:
oh weird, the colors are not rendering. Will look into it

Reply:
Assigning classes dynamically won't work with tailwind. Because tailwind's postcss doesn't know about them at the time of compilation.

If you want to alter the styles dynamically, use the React inline styles API.

Reply:
ah I see. I believe we want to avoid inline style as much as possible but Ill keep that in mind

Reply:
It's weird the blue is loading and the purple is loading on my local machine tho

Reply:
They'll show up if they're statically defined elsewhere. If yes, postcss would have picked it from there.

I see:
- `text-[#8261D2]` is being used in multitenancy: https://github.com/theinfinitereality/eepro-multitenancy/blob/4f28de17320eee91a28e2ced2ab7f75d2b054ed0/components/common/ProjectTitle.tsx#L52-L54
- and  `text-[#375DAF]` is used Filebrowser grid: https://github.com/EtherealEngine/etherealengine/blob/3dd2abcdd75df90a6be24e20e066c4d81841ea64/packages/ui/src/components/editor/panels/Files/browserGrid/index.tsx#L181-L186 Btw, you can use `text-blue-primary` instead of `text-[#375DAF]`
- `A24482` this isn't used anywhere.

Reply:
this is fixed on my machine. let me know if this is still an issue
<img width="429" alt="Screenshot 2024-07-15 at 7 52 50 PM" src="https://github.com/user-attachments/assets/28827146-fd74-46f0-95c8-a93df0fcb54e">